app/models/check.rb

```diff
@@ -14,6 +14,10 @@ def readonly?
     persisted?
   end
 
+  def number_of_days_to_check
```

Comment:
This method name doesn't quite fit to me – since it's on the `Check` model it implies it's the number of days the check took to complete.

Maybe `number_of_days_since_claim_submitted`?

Reply:
👍